core/src/main/java/com/radixdlt/consensus/bft/processor/ProposalTimestampVerifier.java

```diff
@@ -132,6 +139,15 @@ public void processProposal(Proposal proposal) {
     final var prevTimestamp = proposal.getVertex().parentLedgerHeader().proposerTimestamp();
     final var proposalTimestamp = proposal.getVertex().proposerTimestamp();
 
+    metrics
```

Comment:
I wonder if we should filter out `self` here? It should always be near 0 and could skew the metrics if we e.g. average the diffs

Reply:
but I guess we could as well just filter it out when we actually avg :+1:

Reply:
I could.

But we can also calculate `avg_diff_without_self = avg_diff * N/(N-1)`.
(where `N` = number of validators; assuming `self` is near 0 indeed)

Another interpretation of the above fact: the inclusion of `self` here only skews the `avg` metric by a factor of ~`1/N`. It does not skew other interesting metrics, like `max/min`.

And IF the `self`, for some reason, becomes significantly-non-`0`? Then I will appreciate it is in the metrics very much (and will probably go and investigate what causes the delay of proposal processing in the Node internally).